mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,9 +3,16 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
-		var TPL_TASK = '<li>' +
-				'{task}' +
-				'<i class="icon-remove"></i>' +
+		var TPL_FINISHED_TASK = '<li>{taskFinished} ' +
```

Comment:
```{taskFinished}``` is a child node, even if only a text node. As such, it should be indented in new line from its parent.